optaplanner-persistence/optaplanner-persistence-jaxb/src/test/java/org/optaplanner/persistence/jaxb/api/score/buildin/bendable/BendableScoreJaxbXmlJsonAdapterTest.java

```diff
@@ -21,17 +21,30 @@
 
 import org.junit.Test;
 import org.optaplanner.core.api.score.buildin.bendable.BendableScore;
-import org.optaplanner.persistence.jaxb.api.score.AbstractScoreJaxbXmlAdapterTest;
+import org.optaplanner.persistence.jaxb.api.score.AbstractScoreJaxbXmlJsonAdapterTest;
 
-public class BendableScoreJaxbXmlAdapterTest extends AbstractScoreJaxbXmlAdapterTest {
+public class BendableScoreJaxbXmlJsonAdapterTest extends AbstractScoreJaxbXmlJsonAdapterTest {
```

Comment:
By default, a test's name should be <the class it tests> + "Test". By keeping this predictable it's easier to:
1) Find the test for FooBar (it will be called FooBarTest)
2) Avoid that there are 2 test classes for FooBar, which helps to avoid that there are duplicate tests (maintenance issues, less is more, if the coverage is the same)
This test tests BendableScoreJaxbXmlAdapter, so it's name must remain BendableScoreJaxbXmlAdapterTest

Reply:
With the addition of `assertSerializeAndDeserializeJson()` in `AbstractScoreJaxbXmlJsonAdapterTest` the test subclasses now include JSON.

Reply:
I still think the class name shouldn't change because it's a unit test and the unit test class hasn't changed. Let's discuss, I 'll send an invite.